### PR TITLE
feat: added donasi on hamburger menu

### LINKED ISSUE #599: Entry point for Donasi Page
## Description

Related with issue #329 in PR #568, we need to add entry point to the Donasi page after the PR being merged.

1. Add link on hamburger menu
2. Add new icon on the homepage

## Implementation Model

Icon in homepage:

Screen Shot 2021-08-09 at 09 25 27

SVG:

```html
<svg width="20" height="22" viewBox="0 0 20 22" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M11.5807 0.315467L7.43173 5.35885H14.5778L15.9644 3.51009C15.9928 3.47224 16.0245 3.43695 16.0555 3.40115C16.4377 2.95886 16.3345 2.26907 15.8235 1.96247L12.7791 0.135835C12.3881 -0.0987638 11.8857 -0.023459 11.5807 0.315467Z" fill="#1667C2"/>
  <path d="M8.53455 10.6465C8.53455 10.177 8.91511 9.79648 9.38455 9.79648H10.243C10.7124 9.79648 11.093 10.177 11.093 10.6465V12.5049C11.093 13.2114 10.5203 13.7841 9.81377 13.7841C9.10728 13.7841 8.53455 13.2114 8.53455 12.5049V10.6465Z" fill="#1667C2" stroke="#1667C2" stroke-width="0.3"/>
  <path fill-rule="evenodd" clip-rule="evenodd" d="M18.6274 6.5022H1.00045C0.447917 6.5022 0 6.95011 0 7.50265C0 8.05518 0.447917 8.5031 1.00045 8.5031H18.6274C19.18 8.5031 19.6279 8.05518 19.6279 7.50265C19.6279 6.95011 19.18 6.5022 18.6274 6.5022Z" fill="#1667C2"/>
  <path d="M6.24093 9.50369H1.38115C0.828862 9.50369 0.381147 9.9514 0.381147 10.5037V17.5694C0.381147 20.1211 2.70534 22 5.32215 22L14.3058 22C16.9226 22 19.2468 20.1211 19.2468 17.5694V10.5037C19.2468 9.9514 18.7991 9.50369 18.2468 9.50369H13.387C12.9134 9.50369 12.5295 9.88762 12.5295 10.3612C12.5295 10.8348 12.9134 11.2187 13.387 11.2187H16.5317C17.084 11.2187 17.5317 11.6665 17.5317 12.2187V17.5694C17.5317 18.9663 16.1973 20.285 14.3058 20.285L5.32215 20.285C3.43066 20.285 2.0962 18.9663 2.0962 17.5694V12.2187C2.0962 11.6665 2.54392 11.2187 3.0962 11.2187L6.24093 11.2187C6.71453 11.2187 7.09846 10.8348 7.09846 10.3612C7.09846 9.88762 6.71453 9.50369 6.24093 9.50369Z" fill="#1667C2"/>
</svg>

```

On hamburger Menu:

Screen Shot 2021-08-09 at 09 27 02

## Tasks

- [ ] Add new icon on the Homepage
- [ ] Add new link on the Hamburger Menu
- [ ] Add test case to assert new addition

## Figma reference

https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=458%3A1634

### PR DESCRIPTION
**Pull request by ryanadhi**
Closes #599

## Description

Adding menu to reach donation page (`/donasi`)

## Current Tasks

- [x] Add new icon on the Homepage
- [x] Add new link on the Hamburger Menu
- [x] Add test case to assert new addition

### CODE REVIEW

File: __tests__/pages/index.test.tsx [hunk 1]
```diff
@@ -11,10 +11,9 @@ describe("HomePage", () => {
   it("renders the last updated time correctly", () => {
     render(<HomePage latestNews={[latestNewsItemBuilder()]} />);
 
+    expect(screen.getByText(/Pembaruan terakhir pada/i)).toBeVisible();
     expect(
-      screen.getByText(
-        /Pembaruan terakhir pada Selasa, 27 Juli 2021 17.43 WIB/i,
-      ),
+      screen.getByText(/Tuesday, July 27, 2021, 05:43 PM GMT/i),
```
Comment (ryanadhi) [inline __tests__/pages/index.test.tsx:16]: Need to revisit this issue, why the ID test is not passed in my local device, instead it must be using EN format

**Comment by ryanadhi** [comment]
following PR https://github.com/kawalcovid19/wargabantuwarga.com/pull/604
I decided to modify **the last updated test**

**Comment by mazipan** [comment]
Donasi page already on master.

Can we speed up this PR?

**Comment by ryanadhi** [comment]
@mazipan just resolved the conflict, supposed to be ready to be reviewed

**Review by mazipan — APPROVED** [review]
Looks good to me 🚀

**Comment by mazipan** [comment]
@all-contributors please add @ryanadhi for code

**Pull request merged**